hosts.example

```diff
@@ -29,5 +29,14 @@ graphite_fqdn=tendrl.example.com
 #ansible_become=yes
 #ansible_user=cloud-user
 
+# Uncomment to configure etcd to use client to server authentication with HTTPS 
+# client certificates, as well as all Tendrl components will be reconfigured 
+# accordingly.  Default set to false.
+#etcd_tls_client_auth=true
+
+# Uncomment to configure to not use of firewalld on the tendrl_server and 
+# gluster_servers.  Default set to true.
```

Comment:
For this comment to be fully clear, we will need to add much more information there.

Reply:
The readme says "firewalld setup for Tendrl (variable configure_firewalld_for_tendrl)" for the firewalld configuration.

Let me update the comments for the host inventory file and see if the revised version is easier to understand.

Reply:
I've revised this PR -- see https://github.com/Tendrl/tendrl-ansible/pull/110.